src/Bridge/LaravelDoctrine/Extensions/LoggableExtension.php

```diff
@@ -73,12 +74,14 @@ private function getUsernameResolver(): Closure
                 return 'testing';
             }
 
-            if ($this->auth->user() === null) {
+            $user = $this->auth->user();
+
+            if ($user instanceof EntityInterface === false) {
```

Comment:
`instanceof` should be wrapped in parenthesis as per standards
```
if (($user instanceof EntityInterface) === false) {
```

Reply:
done.